google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -318,13 +318,23 @@ def accept(self, visitor, stack=None):
                        'scheduled for deletion']
         stack = [] if not stack else stack
         self._stack = stack
+
+        # Skip the current resource if it's in the excluded_resources list.
+        excluded_resources = visitor.config.variables.get(
+            'excluded_resources', {})
+        resource_name = '{}/{}'.format(self.type(), self.key())
+        if resource_name in excluded_resources:
```

Comment:
I'm sure I missed something, but I'm not seeing how children resources are skipped. Can you please explain how that works?

Reply:
The children resources are added in on line 333, if we skip the current resource here that line won't be called.

Reply:
Got it, i see it now.